tarsier/__main__.py

```diff
@@ -6,14 +6,21 @@
 from playwright.async_api import async_playwright
 
 from tarsier.core import Tarsier
-from tarsier.ocr import GoogleVisionOCRService
+from tarsier.ocr import GoogleVisionOCRService, MicrosoftAzureOCRService
 
 
-async def main(credentials_path: str, url: str, verbose: bool) -> None:
+
+async def main(credentials_path: str, url: str, verbose: bool, ocr_service: str = "google") -> None:
     with open(credentials_path, "r") as f:
         credentials = json.load(f)
 
-    ocr_service = GoogleVisionOCRService(credentials)
+    match ocr_service:
+        case "google":
+            ocr_service = GoogleVisionOCRService(credentials)
+    
+        case "microsoft":
+            ocr_service = MicrosoftAzureOCRService(credentials)
+
```

Comment:
we need to throw an error here as the default case

Reply:
Done